### ISSUE TITLE
Pytorch my model (.pth) is 180M, and the conversion into (.onnx)model is successful, but the conversion into (.onnx)model is 90M, which is reduced by half.

### ISSUE THREAD

**Issue by mojiayuan** [onnx/onnx#3027]

**Comment by askhade** [comment]
@mojiayuan : What is the issue here? Have you tested this model for correctness using onnx checker and ran accuracy tests? It is not necessary that the 2 models be exactly same in size however a difference of half does seem suspicious. Please run checker and accuracy tests and add info here.

**Comment by mojiayuan** [comment]
> @mojiayuan：这是什么问题？您是否使用onnx Checker测试了该模型的正确性并进行了准确性测试？这两个模型的大小不必完全相同，但是相差一半似乎令人怀疑。请运行检查程序和准确性测试，然后在此处添加信息。
Thank you for your reply. I tested the model (719rank1.onnx (90M)), but it was not good. I wonder if something didn't happen during the transformation, so the size of the model was reduced by half.
The project model of https://github.com/songwsx/person_search_demo.    
model Link: https://pan.baidu.com/s/16kyogSsGwL2VgMkNSn9-zg extraction code: f0g9. model file in reID folder --> 719rank1. pth. My code is as follows:
import torch
import torchvision.models as models
from torchvision import transforms as transform
import numpy as np
import torch.onnx
# model = models.vgg11()
#model_resnet = models.resnet50()
import cv2
from reid.data import make_data_loader
from reid.data.transforms import build_transforms
from reid.modeling import build_model
from reid.config import cfg as reidCfg
model_path = "D:/ChromeCoreDownloads/person_search_demo-master/weights/719rank1.pth"

model = build_model(reidCfg, num_classes=10126)
model.load_state_dict(torch.load(model_path, map_location=torch.device('cuda')))
example = torch.ones(1, 3, 256,128).cuda()
model.eval()
model.cuda()
# traced_script_module = torch.jit.trace(reidModel, example)
# Convert to ONNX model
torch.onnx.export(model, example, "719rank1.onnx", verbose=True)
print('Translate end')
